packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -351,12 +351,12 @@ const createRoutes = (
         }
 
         // This call needs to succeed for us to consider this a success.
-        await db.createAccountSubscription(
+        await db.createAccountSubscription({
```

Comment:
Vaguely annoyed I didn't notice this in review, but here's a test to assert it :)

Reply:
Doh!